enhancements/sig-architecture/179-work-feedback-scrape-type/README.md

```diff
@@ -0,0 +1,150 @@
+resources that change frequently and adds overhead from repeated list-and-get operations,
+even when no changes occur. This limits OCM’s ability to provide near-real-time 
+visibility of resource states and can delay downstream reconciliation actions or 
+status reporting in large environments.
+
+By introducing a new field, feedbackScrapeType, OCM allows users and platform operators 
+to choose how feedback is collected—either through the existing periodic polling model 
+(POLL) or an event-driven watch model (WATCH). With WATCH, the managed cluster agent can
+immediately react to changes in specified resources, updating the corresponding 
+ManifestWork and ensuring faster and more efficient feedback propagation to the hub. 
+This reduces the need for fixed polling cadences, minimizes API load, and improves 
+responsiveness for high-value workloads that require timely status updates.
+
+For the ManifestWorkReplicaSet (MWRS) controller, this change ensures it accurately 
+reflects aggregated feedback in real time, responding to both condition and feedback 
+updates from its associated ManifestWork objects. This makes the OCM feedback loop 
+more dynamic, scalable, and adaptive to workload characteristics.
+
+### Goals
+
+* Provide configurable feedback collection modes (POLL or WATCH) per resource in ManifestWork. 
+
+* Ensure MWRS correctly aggregates and reflects feedback updates from both polling and watch-based sources.
+
+* Maintain full backward compatibility for existing users and automation.
+
+### Non-Goals
+
+* Changing the core aggregation semantics in MWRS beyond supporting the new event-driven updates.
+
+## Proposal
+
+This is where we get down to the nitty gritty of what the proposal actually is.
+
+### User Stories
+
+Detail the things that people will be able to do if this is implemented.
+Include as much detail as possible so that people can understand the "how" of
+the system. The goal here is to make this feel real for users without getting
+bogged down.
+
+#### Story 1
+
+User A uses MWRS to deploy a rollout to multiple cluster. The user wants 
+to track the number of pod replicas that contain the change as it is rolled out to the cluster on the MWRS level.
+To do this, the user must get the replica count as the change is rolling out, if it relies on the existing mechanism, it is likely that the change will be missed. 
+
+For example, if the rollout happens every 10 seconds, then User A will not be able to track the per replica count. And only get the count when the rollout is completed.
+
+### Implementation Details/Notes/Constraints [optional]
+
+What are the caveats to the implementation? What are some important details that
+didn't come across above. Go in to as much detail as necessary here. This might
+be a good place to talk about core concepts and how they relate.
+
+### Risks and Mitigation
+
+What are the risks of this proposal and how do we mitigate. Think broadly. For
```

Comment:
I think one of the risks we have to consider is there might be too many watch event, and there could be too many updates call to the manifestwork.

Reply:
Added some more details to this section on how we can limit the updates to the ManifestWork object. 

Need to still think through a few more implementation details (eg. how can MWRS verify if the resource is Ready to stop the informer from watching).